app/assets/onepager/js/send.js

```diff
@@ -326,13 +326,21 @@ function sendTip(email, github_url, from_name, username, amount, comments_public
 
         indicateMetamaskPopup();
         if (isSendingETH) {
-          web3.eth.sendTransaction({
-            from: fromAccount,
-            to: destinationAccount,
-            value: amountInDenom
-          }).once('transactionHash', (txnHash, errors) => {
+          window.ethereum.request({
+            method: 'eth_sendTransaction',
+            params: [
+              {
+                to: destinationAccount,
+                from: fromAccount,
+                value: `${web3.utils.toHex(amountInDenom)}`,
+                gas: `${web3.utils.toHex(318730)}`
```

Comment:
why hardcoding `318730` as the gas limit?

Reply:
@Skalv Can you explain where `318730` comes from?

Reply:
from (your ?) code on web3_modal.js, the default is set to 90000 and all unused gas will return